packages/neos-ui/src/index.js

```diff
@@ -43,7 +43,7 @@ createConsumerApi(manifests, apiExposureMap);
 require('./manifest');
 require('@neos-project/neos-ui-contentrepository');
 require('@neos-project/neos-ui-editors');
-require('@neos-project/neos-ui-views');
+require('@neos-project/neos-ui-views/src/manifest');
```

Comment:
Why is this necessary? Couldn't it be just `require('@neos-project/neos-ui-views');`? Then you'd need to export needed views from `index.js`.

Reply:
That's how it's done for @neos-project/neos-ui-i18n a few lines below that as well. I took that as a reference.

Reply:
Ok, never mind then, could always be refactored.